appvalidator/constants.py

```diff
@@ -44,7 +44,7 @@
         'device-storage:videos', 'device-storage:music',
         'device-storage:sdcard', 'feature-detection',
         'input', 'mobileid', 'mobilenetwork', 'speaker-control', 'systemXHR',
-        'tcp-socket'
+        'tcp-socket', 'nfc'
```

Comment:
Please insert it after `'mobilenetwork'` to keep this sorted (easier to spot stuff added twice this way)